lib/api/response/login_response.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'login_response.g.dart';
+
+@JsonSerializable()
+class LoginResponse {
+  @JsonKey(name: 'access_token')
+  String accessToken;
+  @JsonKey(name: 'token_type')
+  String tokenType;
+  @JsonKey(name: 'expires_in')
+  double expiresIn;
+  @JsonKey(name: 'refresh_token')
+  String refreshToken;
+  @JsonKey(name: 'created_at')
+  double createdAt;
+
+  LoginResponse({
+    required this.accessToken,
+    required this.tokenType,
+    required this.expiresIn,
+    required this.refreshToken,
+    required this.createdAt,
+  });
+
+  factory LoginResponse.fromJson(Map<String, dynamic> json) =>
+      _$LoginResponseFromJson(json);
```

Comment:
I don't think we can parse the `json:api` format easily like this as the login response is included under a `data` node. Some of samples to checkout to resolve this:
- https://github.com/nimblehq/internal-certification-flutter/blob/main/manh/lib/api/response/base/base_http_response.dart#L11 👉 this seems to be a workaround
- https://github.com/luongvo/flutter-survey/blob/develop/lib/api/response/oauth_token_response.dart#L22-L23 👉 this is a more elegant way as it uses pre-built json-api [japx](https://pub.dev/packages/japx) lib to parse json:api data structure.